tests/infra_test.go

```diff
@@ -490,11 +490,11 @@ var _ = Describe("Infrastructure", func() {
 			defer expecter.Close()
 
```

Comment:
looks like commit message has no first line so it appears weird. can you add a first line to the commit?

Reply:
@RamLavi  sorry for the mess. let's move the discussion here: https://github.com/kubevirt/kubevirt/pull/4010